### PR TITLE
Feature/update userdetails config

### PR DESCRIPTION
**Pull request by yasima-csiro**

### CODE REVIEW

File: ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy [hunk 1]
```diff
@@ -33,8 +33,8 @@ class UserListService {
     }
 
     private void checkConfig() {
-        if (!grailsApplication.config.getProperty('userDetails.url')) {
```
Comment (sbearcsiro) [inline ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy:36]: Can we have these fall back to the old `userDetails.url` if the new property isn't set?

**Review by sbearcsiro — COMMENTED** [review]
Can we add a fallback to the previous setting?  Possibly with a logged warning to update to the new property.

File: ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy (hunk 1 above)
Comment (yasima-csiro) [reply ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy:36]: sure

Comment (yasima-csiro) [reply ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy:36]: done

**Review by sbearcsiro — APPROVED** [review]

**Pull request merged**